include/veriblock/blob.hpp

```diff
@@ -40,8 +40,11 @@ struct Blob {
     data_.fill(0);
     assign(str);
   }
+  Blob(Blob<N>& other) {
+    data_.fill(0);
```

Comment:
no need to do this before `assign`

Reply:
Will it properly assign smaller size blob?

Reply:
Hm. I don't think so.